src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/PlayerDeathSystem.java

```diff
@@ -65,6 +66,8 @@ public void beforeDestroy(BeforeDestroyEvent event, EntityRef player, CharacterC
             player.send(new RestoreFullHealthEvent(player));
             Vector3f randomVector = new Vector3f(-1 + random.nextInt(3), 0, -1 + random.nextInt(3));
             player.send(new CharacterTeleportEvent(randomVector.add(LASUtils.getTeleportDestination(team))));
+            float playerYaw = team.equals(LASUtils.RED_TEAM) ? -90 : 90;
```

Comment:
While this works currently for our simple map, this code would be more resilient if we'd compute the yaw from the teleport target position and the opponent player base position.

Reply:
Made use of this https://stackoverflow.com/questions/5188561/signed-angle-between-two-3d-vectors-with-same-origin-within-the-same-plane as the default calculation of angles isn't signed.